openstef/feature_engineering/missing_values_transformer.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-FileCopyrightText: 2017-2023 Contributors to the OpenSTEF project <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from typing import Union, List
+
+import numpy as np
+import pandas as pd
+from sklearn.impute import SimpleImputer
+from sklearn.preprocessing import FunctionTransformer
+from sklearn.utils.validation import check_array
+
+
+class MissingValuesTransformer:
+    in_feature_names: List[str] = None
```

Comment:
Maybe a small docstring where someone can use this class for would be helpful:)

Reply:
Nice suggestions. Fixed!